changelog.md

```diff
@@ -1,10 +1,17 @@
 # Changelog
 
+## Unreleased
+
+### Features
+
+- [#3536](https://github.com/ignite/cli/pull/3536) Change app.go to v2 and add AppWiring feature
```

Comment:
This is included in the main changelog, but this PR hasn't been merged. Is it expected?

Reply:
Found the one: https://github.com/ignite/cli/pull/3553, it should be this one.